docs/4.3/kubernetes_ssh.md

```diff
@@ -237,10 +294,47 @@ $ tctl create -f admin.yaml
     Teleport 4.3 has an option to extract the local part from an email claim. This can be helpful
     since some operating systems don't support the @ symbol. This means by using `logins: ['{% raw %}{{email.local(external.email)}}{% endraw %}']` the resulting output will be `dave.smith` if the email was dave.smith@acme.com.
 
-Once this is complete, when users execute `tsh login` and go through the usual Okta login
+Once setup is complete, when users execute `tsh login` and go through the usual Okta login
 sequence, their `kubeconfig` will be updated with their Kubernetes credentials.
 
 !!! note
 
     For more information on integrating Teleport with Okta, please see the
     [Okta integration guide](enterprise/sso/ssh_okta.md).
+
+## AWS EKS
+
+We've a complete guide on setting up Teleport with EKS. Please see the [Using Teleport with EKS Guide](aws_oss_guide.md#using-teleport-with-eks).
+
+## Multiple Kubernetes Clusters
+
+You can take advantage of the [Trusted Clusters](trustedclusters.md) feature of
+Teleport to federate trust across multiple Kubernetes clusters.
+
+When multiple trusted clusters are present behind a Teleport proxy, the
+`kubeconfig` generated by [ `tsh login` ](cli-docs.md#tsh-login) will contain the
+Kubernetes API endpoint determined by the `<cluster>` argument to [`tsh
+login`](cli-docs.md#tsh-login) .
+
+* There are three Teleport/Kubernetes clusters: "main", "east" and "west". These
```

Comment:
Nit: before this, add something like "For example, consider the following setup:"